CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+Change Log
+==========
+
+1.3
+---
+* Change FxATestAccount constructor to accept the url to the FxA API server
+  * This is a **breaking change**
+
+1.2
+---
+* Update required version of PyFxA in setup.py
+
+1.1
+---
+* Update required version of PyFxA in requirements.txt
```

Comment:
Do we need a change log? If so, I think the best place for this to be listed is PyPI but this file isn't currently displayed there. Probably a future enhancement but I'd recommend switching the README to be displayed on PyPI and include the change log there. See an example of pulling in the README contents for PyPI here: https://github.com/davehunt/pytest-html/blob/fa3cf426036a2cdc3a5b87e7e604937bc0ee88a2/setup.py#L6